activestorage/test/template/image_tag_test.rb

```diff
@@ -0,0 +1,40 @@
+require "test_helper"
+require "database/setup"
+
+class User < ActiveRecord::Base
+  has_one_attached :avatar
+end
+
+class ActiveStorage::ImageTagTest < ActionView::TestCase
```

Comment:
This entire file is coupling ActiveStorage to ActionView that is not a dependency of activestorage. We need to move those tests to actionview in the same way we do with activerecord.